submissions/DeamonFire/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="css/style.css">
+        <title>Hooli-style Popup</title>
+    </head>
+    <body>
+        <header class="header">
+            <h1 class="header__h1">Hooli-style Popup</h1>
+            <nav class="header__nav">
+                <ul class="header__menu">
```

Comment:
Avoid shifting sibling elements on hover or focus. [Watch video](https://share.vidyard.com/watch/8oDnqwEyGidcU2frYpoxJ9?)
Also, check the hit-box of your elements: when the user hovers `li` element you display the border but the cursor is the default, and you change it when the users hover the link inside `li`. All changes on hover must be at the same time.

Reply:
Solved this problem =)